distribution/docs/src/main/resources/_contents/_action-framework/developing-action-components-contents.adoc

```diff
@@ -0,0 +1,62 @@
+
+==== Developing Action Components
+
+To provide a service, such as a link to a record, the `ActionProvider` interface should be implemented.
+An `ActionProvider` essentially provides a List of ``Action``s when given input that it can recognize and handle.
+For instance, if a REST endpoint ActionProvider was given a metacard, it could provide a link based on the metacard's ID. 
+An Action Provider performs an action when given a subject that it understands.
+If it does not understand the subject or does not know how to handle the given input, it will return `Collections.emptyList()`.
+An Action Provider is required to have an ActionProvider id.
+The Action Provider must register itself in the OSGi Service Registry with the `${ddf-branding-lowercase}.action.ActionProvider` interface and must also have a service property value for `id`. 
+An action is a URL that, when invoked, provides a resource or executes intended business logic. 
+
+===== Action Component Naming Convention
+
+For each Action, a title and description should be provided to describe what the action does.
+The recommended naming convention is to use the verb 'Get' when retrieving a portion of the metacard, such as the metadata or thumbnail, or when you are downloading the product.
+The verb 'Export' or expression 'Export as' is recommended when the metacard is bring exported in a different format or presented after going some transformation.
```

Comment:
Agreed with above. Should also add an extra `the` prior to `expression` to clarify your `or` is between two concrete things: 
```
The verb `Export` or the expression `Export as` is recommended ...
```